### PR TITLE
Post method

### PR DESCRIPTION
**Pull request by Carolina-Vacaflor**
Hi, please review my code.

### CODE REVIEW

File: .env
Comment (celestepalet) [inline .env:1]: add a line empty at the end. And you need add the variable for endpoint url

File: .gitignore
Comment (celestepalet) [inline .gitignore:1]: please add a empty line at the end of the file

File: libraries/format_response.py
Comment (celestepalet) [inline libraries/format_response.py:1]: Please add copyright and method comments, aldo delete comments lines

File: libraries/get_url.py
Comment (celestepalet) [inline libraries/get_url.py:1]: Please add copyright and method comments

File: libraries/make_request.py
Comment (celestepalet) [inline libraries/make_request.py:1]: Please add copyright and method comments, delete method that you did not implement

File: libraries/validation.py
Comment (celestepalet) [inline libraries/validation.py:1]: Please add copyright and method comments and delete the method non implemented

File: results/output.xml
Comment (celestepalet) [inline results/output.xml:1]: do not upload the results files

**Comment by celestepalet** [comment]
Please solve conflicts and made tha changes required, regards!

**Review by celestepalet — CHANGES_REQUESTED** [review]
Please solve conflicts and made tha changes required, regards!

File: .env
Comment (Carolina-Vacaflor) [reply .env:1]: Done!

File: .gitignore
Comment (Carolina-Vacaflor) [reply .gitignore:1]: Done!

File: libraries/format_response.py
Comment (Carolina-Vacaflor) [reply libraries/format_response.py:1]: Done!

File: libraries/get_url.py
Comment (Carolina-Vacaflor) [reply libraries/get_url.py:1]: Done!

File: libraries/make_request.py
Comment (Carolina-Vacaflor) [reply libraries/make_request.py:1]: Done

File: libraries/validation.py
Comment (Carolina-Vacaflor) [reply libraries/validation.py:1]: Done!

File: results/output.xml
Comment (Carolina-Vacaflor) [reply results/output.xml:1]: Done!

**Pull request merged**